src/commands/test.ts

```diff
@@ -104,6 +104,11 @@ export class TestCommand implements Command {
       description: "Set job visibility to 'public', 'public restricted', 'share', 'team' or 'private'",
       type: String,
     },
+    {
+      name: 'configFile',
```

Comment:
all cli flags have followed the `dash-case` format, please change to `config-file` and handle the mapping into WCT below.